pkg/controller/operators/catalog/operator.go

```diff
@@ -551,6 +554,84 @@ func (o *Operator) ExecutePlan(plan *v1alpha1.InstallPlan) error {
 					plan.Status.Plan[i].Status = v1alpha1.StepStatusCreated
 				}
 
+			case roleKind:
```

Comment:
Does this cause problems with [the logic in the `olm-operator`](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/pkg/controller/install/deployment.go#L68) that creates RBAC resources?

Reply:
Good catch. I was planning on changing the [`olm-operator` logic](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/pkg/controller/install/deployment.go#L68) to handle only RBAC verification in a follow up, but it's probably better to go ahead and handle that now.